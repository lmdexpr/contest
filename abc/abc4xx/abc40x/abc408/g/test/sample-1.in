5
3 2 2 1
5 2 8 3
1 2 2 1
60 191 11 35
40 191 71 226
