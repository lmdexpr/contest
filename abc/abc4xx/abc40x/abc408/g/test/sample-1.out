3
5
1
226
4
